schema/ext/vulnerability-1.0.xsd

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:vc="http://www.w3.org/2007/XMLSchema-versioning"
+           xmlns:dg="http://cyclonedx.org/schema/ext/vulnerability/1.0"
+           elementFormDefault="qualified"
+           targetNamespace="http://cyclonedx.org/schema/ext/vulnerability/1.0"
+           vc:minVersion="1.0"
+           vc:maxVersion="1.1"
+           version="1.0.0-SNAPSHOT">
+
+    <xs:complexType name="cvssV2">
+        <xs:annotation>
+            <xs:documentation>
+                Defines the structure of a CVSS v2 vulnerability
+                https://www.first.org/cvss/v2/guide
+            </xs:documentation>
+        </xs:annotation>
+        <xs:sequence>
+            <xs:element name="score" type="xs:decimal" minOccurs="1" maxOccurs="1"/>
+            <xs:element name="attackVector" type="xs:string" minOccurs="1" maxOccurs="1"/>
+            <xs:element name="accessComplexity" type="xs:string" minOccurs="1" maxOccurs="1"/>
+            <xs:element name="authentication" type="xs:string" minOccurs="1" maxOccurs="1"/>
+            <xs:element name="confidentialImpact" type="xs:string" minOccurs="1" maxOccurs="1"/>
+            <xs:element name="integrityImpact" type="xs:string" minOccurs="1" maxOccurs="1"/>
+            <xs:element name="availabilityImpact" type="xs:string" minOccurs="1" maxOccurs="1"/>
+            <xs:element name="severity" type="xs:string" minOccurs="1" maxOccurs="1"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="cvssV3">
```

Comment:
For vulnerabilities that reside in a public source (e.g. NVD), the base score attributes are generally good enough. However, without temporal or environmental attributes, CVSS would not allow an organization to fine tune the severity based on their environment.

Reply:
> For vulnerabilities that reside in a public source (e.g. NVD), the base score attributes are generally good enough. However, without temporal or environmental attributes, CVSS would not allow an organization to fine tune the severity based on their environment.

👍 Taking above in to consideration and  https://github.com/CycloneDX/specification/pull/19#discussion_r320037371 restructured the scoreType to be generic that can support multiple risk rating standards. Most of the elements have been made optional.